apis/core/validation/blueprint.go

```diff
@@ -62,6 +73,31 @@ func validateBlueprintImportDefinitions(fldPath *field.Path, imports []core.Impo
 			importNames.Insert(importDef.Name)
 		}
 
+		if len(importDef.Type) != 0 {
+			// type is specified, use new validation
+			switch importDef.Type {
+			case core.ImportTypeData:
+				if importDef.Schema == nil {
```

Comment:
currently we ignore other fields.
Maybe we should check if other fields are defined that do not belong to the current type.